docs/migration-guide/faq.md

```diff
@@ -0,0 +1,7 @@
+# FAQ
+
+**Is `spatial local worker launch` supported?**<br/>
```

Comment:
```suggestion
**Do you support `spatial local worker launch`?**<br/>
```
Passive -> active. Let me know if it's OK to say "you" here - hopefully it should be fairly clear what it means from the context.

Reply:
Nice! Done. :)